Purchases/RCUserManager.h

```diff
@@ -0,0 +1,37 @@
+//
+// Created by RevenueCat.
+// Copyright (c) 2019 Purchases. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "RCDeviceCache.h"
+
+@class RCPurchaserInfo, RCBackend;
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface RCUserManager : NSObject
```

Comment:
Had to rename this in Android to IdentityManager because there's a UserManager already, what do you think about that name @jeiting ?

Reply:
both fine

Reply:
ok, I will change it here to match Android